### PR TITLE
Jimin Han pull request

### PR DESCRIPTION
**Pull request by hanjimin06**
Jimin's pull request. I edited my README as the homework required.

### CODE REVIEW

**Review by Wendychenn — COMMENTED** [review]
Hi Jimin,
        You've done a good job on this HW01!
        Firstly, I find you use “\” at the end of the paragraph but not leave a blank paragraph between the two which is awesome.  And the fact that you don’t use link to represent your image but a local file instead shows you really explore yourself to use R Markdown. The whole arrangement is very easy to read, so I want to give you the point!
       However, there is a conflict in your list: after you said difficulties, you mentioned forking is easier than you thought. This should be avoided in order to make whole pages more logical. Six difficulties are too many to read, if I were you, I will summarize them to 3-4 rows. After looking at all the commits, I suggest you check the content before you’re ready to push so that it will not create too many history files.  

 Wen
 
## Evaluation 

Topic | Excellent | Satisfactory | Needs Improvement
-- | -- | -- | --
Coding style | ⭕  |  |  
Coding strategy | ⭕ |  |  
Presentation |  |   |  ⭕
Achievement, creativity |  |  ⭕ |  
Reproducibility | ⭕ |   |

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# hw01
+## Jimin Han 101
+My name is **Jimin Han**. \
+I am from Korea. *Yes, South not North*.\
```
Comment (bensoltoff) [inline README.md:4]: If you want line breaks, you can also just leave a blank line between each sentence.

```
My name is **Jimin Han**.

I am from Korea. *Yes, South not North*.
```

My name is **Jimin Han**.

I am from Korea. *Yes, South not North*.

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
+[Here](https://www.instagram.com/jiminhan_photography/) are my photos.  
+I hope to become proficient in R after taking this class. 
+
+At first there were some difficulties in doing this assignment: 
+1. Forking the HW (Easier than I thought). 
+2. Getting the correct directories. 
+3. Familiarizing myself with how computers work 
+4. My branch was ahead of my master file, so I had to redo. 
+5. I hope in the course of few weeks I'll be able to breeze through these 'minor' inconveniences. 
+6.The biggest part was commiting the images. I thought I had done everything right, but as a matter of fact I had not commited the image file itself. That took quite a long time to fix.
```
Comment (bensoltoff) [inline README.md:15]: You can just leave everything numbered as 1. When you render the Markdown document each item is automatically numbered.

```
1. Forking the HW (Easier than I thought). 
1. Getting the correct directories. 
1. Familiarizing myself with how computers work 
1. My branch was ahead of my master file, so I had to redo. 
1. I hope in the course of few weeks I'll be able to breeze through these 'minor' inconveniences. 
1. The biggest part was commiting the images. I thought I had done everything right, but as a matter of fact I had not commited the image file itself. That took quite a long time to fix. 
```

1. Forking the HW (Easier than I thought). 
1. Getting the correct directories. 
1. Familiarizing myself with how computers work 
1. My branch was ahead of my master file, so I had to redo. 
1. I hope in the course of few weeks I'll be able to breeze through these 'minor' inconveniences. 
1. The biggest part was commiting the images. I thought I had done everything right, but as a matter of fact I had not commited the image file itself. That took quite a long time to fix.

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+1. Forking the HW (Easier than I thought). 
+2. Getting the correct directories. 
+3. Familiarizing myself with how computers work 
+4. My branch was ahead of my master file, so I had to redo. 
+5. I hope in the course of few weeks I'll be able to breeze through these 'minor' inconveniences. 
+6.The biggest part was commiting the images. I thought I had done everything right, but as a matter of fact I had not commited the image file itself. That took quite a long time to fix. 
+
+But with help from stack exchange and the TAs, I was able to slowly do all the things listed in the HW. It was a satisfying procedure.
+
+![Random Photo](Images/IMG_5383.JPG)
```
Comment (bensoltoff) [inline README.md:19]: Nice embedding of an image. Good use of the repo to publish it online

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |  x         |              |            |
| **Presentation**            |           |   x           |            |
| **Achievement, creativity** |           |    x          |            |
| **Reproducibility**         |   x        |              |            |

**Pull request closed without merging**